easyDiffractionApp/Gui/Components/UserTutorialsController.qml

```diff
@@ -36,9 +36,9 @@ EaElements.RemoteController {
         interval: 1000
         onTriggered: {
             startScreenRecording()
-            runDataFittingTutorial()
-            //runDataSimulationTutorial()
-            //stopScreenRecording()
+            //runDataFittingTutorial()
+            runDataSimulationTutorial()
+            stopScreenRecording()
         }
```

Comment:
It would be useful to see why certain tutorials are disabled - can you please add a comment or a TODO?